LethalCompanyVR.csproj

```diff
@@ -79,9 +78,6 @@
   </ItemGroup>
 
   <Target Name="PostBuild" AfterTargets="PostBuildEvent" Condition=" '$(OS)' == 'Windows_NT' ">
-    <Copy SourceFiles="$(OutDir)LethalCompanyVR.dll"
-      DestinationFolder="C:\Program Files (x86)\Steam\steamapps\common\Lethal Company\BepInEx\plugins\"
-      SkipUnchangedFiles="true"
-    />
+    <Copy SourceFiles="$(OutDir)LethalCompanyVR.dll" DestinationFolder="C:\Program Files (x86)\Steam\steamapps\common\Lethal Company\BepInEx\plugins\" SkipUnchangedFiles="true" />
```

Comment:
I'm not so sure about this. While it's convenient, not everybody has LC installed on C drive. Perhaps don't push that change, but keep it locally because it is _really_ convenient.

Reply:
^ I second this. A lot of players will be uninstalling the game (and reinstalling it) depending on their rotating game library.

Reply:
That's just the build thing in a template I use, I don't know how to take it out in pr